src/test/java/ca/corefacility/bioinformatics/irida/ria/integration/analysis/AnalysisDetailsPageIT.java

```diff
@@ -136,7 +136,7 @@ public void testOutputFiles() {
 				page.downloadOutputFileButtonVisible(2));
 
 		// Has no output files
-		page = AnalysisDetailsPage.initPage(driver(), 10L, "output");
+		page = AnalysisDetailsPage.initPage(driver(), 10L, "");
```

Comment:
We should change this back to `output` since my comments above will allow this to still work as before.

Reply:
Updated in 9da8e9f